Source/Bifrost.Web/Bifrost.Web.csproj

```diff
@@ -212,8 +214,18 @@
     <EmbeddedResource Include="Scripts\knockout-3.4.0.debug.js" />
     <EmbeddedResource Include="Scripts\knockout-3.4.0.js" />
   </ItemGroup>
-  <Import Project="..\Solutions\packages\LessMsbuildTasks.1.0.7\tools\LessCompiler.targets" Condition="Exists('..\Solutions\packages\LessMsbuildTasks.1.0.7\tools\LessCompiler.targets')" />
+  <ItemGroup>
+    <Content Include="Scripts\jquery.signalR-2.2.1.js" />
```

Comment:
Keep these as an embedded resource. You must also update the file name in ApplicationHttpHandler.